### ISSUE TITLE
do not modify body style attribute

### ISSUE THREAD

**Issue by landsman** [emilkowalski/vaul#324]
Please add an option to do not modify body style attribute when drawer is opened. 

Currently there are these styles:

```css
position: fixed !important;
top: 0px;
left: 0px;
right: 0px;
height: auto;
pointer-events: none;
```

_Code: https://github.com/emilkowalski/vaul/blob/main/src/use-position-fixed.ts#L34-L38_

This is a problem when you use more portals, for example, modal above the drawer (z-index does not work properly, it's not clickable because of the overlay).

Especialy `pointer-events` is the problem here.

I do manage this via my custom attribute (`data-scroll-disabled`) which I am able to change when drawer is opened on my own.

**Comment by aaronleopold** [comment]
Even the ability to modify any of the attributes and/or add additional would be greatly appreciated. E.g. setting height to `100%` instead of `auto`.

**Comment by artemis-prime** [comment]
+1 This breaks any other fw or tool that uses the body tag.  Let's find a better, less brittle way.

**Comment by manylovv** [comment]
+1

**Comment by emilkowalski** [comment]
Added `noBodyStyles` prop! It's now available in `0.9.1`

noBodyStyles: When true the body doesn't get any styles assigned from Vaul.

**Comment by pinktonio** [comment]
`pointer-events: none` is still assigned even when `noBodyStyles={true}` is set.

**Comment by emilkowalski** [comment]
This is fixed in #409
